crates/bevy_widget/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+mod widget;
```

Comment:
Can you add forbid(unsafe_code) and warn(missing_docs) crate level rules please?

Once that's in, we should add basic doc strings.

Reply:
The only thing with missing docs is `QueuedText`. It's only used as a `Local` for `text_system`, so should we even keep it public?

Reply:
We should probably keep text_system public, so we need to keep the Local type public or Rust will complain.